base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ScriptGeneratorViewModel.java

```diff
@@ -1031,6 +1033,54 @@ public void update(ViewerCell cell) {
     
         });
         timeEstimateColumn.getColumn().setAlignment(SWT.CENTER);
+        
+        // Add estimated custom columns.
```

Comment:
This method is getting far too long to be readable/understandable. Can you refactor out some helper functions?